frontend/src/components/UserInfo/UserAvatarWithInfo.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Redirect } from 'react-router';
+import {withStyles} from '@material-ui/core/styles';
+import UserInfo from "./UserInfo";
+import UserAvatar from "./UserAvatar";
+import {redirectToSignUp} from "../../Service/NeedAuthorization";
+
+const styles = {
+    forDiv: {
+        display: "flex",
+        justifyContent: "center",
+    },
+};
+
+class UserAvatarWithInfo extends React.Component {
+
+    state = {
+        userInfo: [],
+        success: false,
+        error: "",
+        token: "Peter P",
+    };
+
+    componentDidMount() {
+
+        const headers = new Headers({
+            'Content-Type': 'application/json',
+            'Authorization': this.state.token
+        });
+
+
+        fetch('http://localhost:6543/profile', {method: "GET", headers})
+            .then(response => response.json())
+            .then(data => this.setState({userInfo: data.data[0], success: data.success, error: data.error}))
+            .catch(err=>console.log(err))
```

Comment:
Most of users don't know nothing about browser console. You need to let them now, that something went wrong.